pkg/apis/extensions/v1beta1/register.go

```diff
@@ -60,6 +58,7 @@ func addKnownTypes() {
 		&ListOptions{},
 		&ConfigMap{},
 		&ConfigMapList{},
+		&v1.DeleteOptions{},
```

Comment:
This makes the scheme map `extensions,v1beta1,DeleteOptions` maps to `v1.DeleteOptions`. Why would I want that to happen?  I'd rather have code that expects a `v1.DeleteOptions` decode it with the proper scheme.  Why would would I want to cross register it?